modules/test/playwright/pages/product-navigation-control-menu-web/ProductMenuPage.ts

```diff
@@ -72,7 +72,7 @@ export class ProductMenuPage {
 
 	async clickSpecificPage(pageName: string) {
 		await this.pagesButton.click();
-		await this.page.getByLabel(pageName, {exact: true}).click();
+		await this.page.getByLabel(pageName, {exact: true}).first().click();
```

Comment:
Why is this `.first()` needed? In theory we are creating pages with randomString, and with the `{exact: true}` we shouldn't have more than one page with the same name.

Reply:
without isolatedSiteTest i was having problems with the not deleted ones. All sorted with that, and i dont need to use. Thank you!